inc/TRestRawSignalAnalysisProcess.h

```diff
@@ -42,6 +42,9 @@ class TRestRawSignalAnalysisProcess : public TRestEventProcess {
     /// The range where the observables will be calculated
     TVector2 fIntegralRange = TVector2(10, 500);
 
+    /// Option fo calculation of baseline parameters, can be set to "ROBUST"
+    std::string fBaseLineOption = "";
```

Comment:
```suggestion
    std::string fBaseLineOption = ""; //<
```

Reply:
Better to specify that the metadata member will be part of the streamer, i.e. stored in disk. This is the default anyhow.